app/js/lib/foxbox/recipes.js

```diff
@@ -173,6 +187,16 @@ export default class Recipe {
               ];
               break;
 
+            case 'ip-camera@link.mozilla.org':
+              service.setters = [
+                {
+                  label: 'takes a picture',
+                  value: 'setter:snapshot.ae67e622-7a66-465e-bab0-' +
```

Comment:
ugh, hardcoding such things is bad, but let's move forward with this and will change after this big demo :/ Anyway I will be poking around this piece of code right now.